packages/extension/src/companion/CompanionContextMenu.tsx

```diff
@@ -90,7 +91,10 @@ export default function CompanionContextMenu({
         <Item onClick={onBookmark}>
           <BookmarkIcon
             size="medium"
-            className="mr-2"
+            className={classNames(
+              'mr-2',
+              postData?.bookmarked && 'text-theme-color-bun',
+            )}
```

Comment:
Is this a merge conflict?
We should aim to get this live on the main branch before squads if possible.

Reply:
I think so too. I just pulled main and this seems to be gone